src/table/interfaces.tsx

```diff
@@ -228,6 +228,22 @@ export interface TableProps<T = any> extends BaseComponentProps {
    * @visualrefresh `embedded`, `stacked`, and `full-page` variants
    */
   variant?: TableProps.Variant;
+
+  /**
+   * Use this property to inform screen readers how many items there are in a table.
+   * It specifies the total count of all items in a table.
+   * If there is an unknown total of items in a table, leave this property undefined.   */
+  totalItemsCount?: number;
+  /**
+   *  Use this property to inform screen readers which range of items is currently displayed in the table.
+   *  It specifies the index (1-based) of the first item in the table.
+   *  If the table has no pagination, leave this property undefined.   */
+  firstIndex?: number;
+  /**
+   * Use this function to announce page changes for screen reader users.
+   * When firstIndex is available, it returns a text when the firstIndex or totalItemsCount change.
+   */
+  liveAnnouncement?: (data: TableProps.LiveAnnouncement) => string;
```

Comment:
How do we name ARIA live related properties in other components? I think we usually include `aria` in there somewhere.

Reply:
I didn't find another place where we have a property that is only used in aria-live

Reply:
We have `renderHighlightedAriaLive` for Autosuggest, Select, Multiselect, and `renderSelectedAbsoluteRangeAriaLive` for Date range picker.